### ISSUE TITLE
Conversion failed on reinstall CL-MariaDB

### ISSUE THREAD

**Issue by cloud86robb3rt** [plesk/cloudlinux7to8#9]
After the reboot the stage 'Update databases' failed with the reinstall of MariaDB;

`cloudlinux7to8 process has failed. Error: Failed: update modern mariadb installed from governor. The reason: Command '['dnf', 'module', 'enable', 'mariadb:cl-MariaDB1011']' returned non-zero exit status 1.`

The conversion itself succeed, the server came back online with CloudLinux 8, however it seems that the last part of the process failed on MariaDB.

```
2024-07-29 17:56:17,239 - INFO - Start stage 'Update databases'.
2024-07-29 17:56:17,240 - INFO - Skipped: install mysql connector
2024-07-29 17:56:17,240 - DEBUG - Invoking action 'update modern mariadb installed from governor'
2024-07-29 17:56:17,240 - INFO - Do: update modern mariadb installed from governor
2024-07-29 17:56:17,241 - DEBUG - Adopt repofile '/etc/yum.repos.d/cl-mysql.repo'
2024-07-29 17:56:17,242 - DEBUG - Repository file line: [cl-mysql-meta]
2024-07-29 17:56:17,242 - DEBUG - Repository file line: name=cl-mysql
2024-07-29 17:56:17,242 - DEBUG - Repository file line: baseurl=http://repo.cloudlinux.com/other/cl$releasever/mysqlmeta/cl-mariadb-10.4/$basearch/
2024-07-29 17:56:17,242 - DEBUG - Repository file line: enabled=1
2024-07-29 17:56:17,242 - DEBUG - Repository file line: gpgcheck=1
2024-07-29 17:56:17,242 - DEBUG - Repository file line: gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CloudLinux
2024-07-29 17:56:17,242 - DEBUG - Repository file line:
2024-07-29 17:56:17,242 - DEBUG - Adopt repository with id 'cl-mysql-meta' is extracted.
2024-07-29 17:56:17,243 - DEBUG - Repository file line: [mysqclient]
2024-07-29 17:56:17,243 - DEBUG - Repository file line: name=mysqlclient
2024-07-29 17:56:17,243 - DEBUG - Repository file line: baseurl=http://repo.cloudlinux.com/other/cl$releasever/mysqlmeta/mysqlclient/$basearch/
2024-07-29 17:56:17,243 - DEBUG - Repository file line: enabled=0
2024-07-29 17:56:17,243 - DEBUG - Repository file line: gpgcheck=1
2024-07-29 17:56:17,243 - DEBUG - Repository file line: gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CloudLinux
2024-07-29 17:56:17,243 - DEBUG - Adopt repository with id 'mysqclient' is extracted.
2024-07-29 17:56:18,954 - DEBUG - Detected mariadb version is: 10.11.8
2024-07-29 17:56:18,955 - DEBUG - Going to reinstall following packages with enabled dnf module 'mariadb:cl-MariaDB1011': ['cl-MariaDB1011-server', 'cl-MariaDB-meta', 'cl-MariaDB1011', 'cl-MariaDB-meta-devel', 'cl-MariaDB-meta-client', 'cl-MariaDB1011-devel', 'cl-MariaDB1011-libs']
2024-07-29 17:56:41,617 - INFO - Running: ['/usr/bin/rpm', '-e', '--nodeps', 'cl-MariaDB1011-server', 'cl-MariaDB-meta', 'cl-MariaDB1011', 'cl-MariaDB-meta-devel', 'cl-MariaDB-meta-client', 'cl-MariaDB1011-devel', 'cl-MariaDB1011-libs']. Output:
2024-07-29 17:56:42,793 - INFO - warning: /etc/my.cnf saved as /etc/my.cnf.rpmsave
2024-07-29 17:56:42,911 - INFO - Command ['/usr/bin/rpm', '-e', '--nodeps', 'cl-MariaDB1011-server', 'cl-MariaDB-meta', 'cl-MariaDB1011', 'cl-MariaDB-meta-devel', 'cl-MariaDB-meta-client', 'cl-MariaDB1011-devel', 'cl-MariaDB1011-libs'] finished successfully
2024-07-29 17:56:42,911 - INFO - Running: ['dnf', 'module', 'enable', 'mariadb:cl-MariaDB1011']. Output:
2024-07-29 17:56:44,442 - INFO - This system is receiving updates from CloudLinux Network server.
2024-07-29 17:56:45,334 - INFO - cloudlinux-x86_64-server-8                       11 kB/s | 4.2 kB     00:00
2024-07-29 17:56:46,798 - INFO - Error: Problems in request:
2024-07-29 17:56:46,798 - INFO - missing groups or modules: mariadb:cl-MariaDB1011
2024-07-29 17:56:46,851 - ERROR - Command ['dnf', 'module', 'enable', 'mariadb:cl-MariaDB1011'] failed with return code 1
2024-07-29 17:56:46,851 - ERROR - Failed: update modern mariadb installed from governor. The reason: Command '['dnf', 'module', 'enable', 'mariadb:cl-MariaDB1011']' returned non-zero exit status 1.

2024-07-29 17:56:47,074 - DEBUG - Sent error report
2024-07-29 17:56:47,083 - DEBUG - Trying to send status of conversion by report-update utility '/var/cache/parallels_installer/report-update'
2024-07-29 17:56:47,724 - ERROR - cloudlinux7to8 process has failed. Error: Failed: update modern mariadb installed from governor. The reason: Command '['dnf', 'module', 'enable', 'mariadb:cl-MariaDB1011']' returned non-zero exit status 
1.
```
The feedback log is attached
[cloudlinux7to8_feedback.zip](https://github.com/user-attachments/files/16416254/cloudlinux7to8_feedback.zip)
.

**Comment by cloud86robb3rt** [comment]
After running CloudLinux SQL Governor installation procedure; https://docs.cloudlinux.com/legacy/cloudlinux_os_components/#installation, it successfully installed MariaDB and after a reboot continued with the conversion.

**Comment by SandakovMM** [comment]
Problem should be fixed by #11
